tests/python/pants_test/engine/legacy/test_build_ignore_integration.py

```diff
@@ -20,55 +20,40 @@ def use_pantsd_env_var(cls):
     return False
 
   def test_build_ignore_patterns_pants_ini(self):
+    target_path = "testprojects/src/java/org/pantsbuild/testproject/phrases"
+    targets = [
+      f"{target_path}:{target}"
+      for target in ["ten-thousand", "once-upon-a-time", "lesser-of-two", "there-was-a-duck"]
+    ]
+
     def output_to_list(output_filename):
       with open(output_filename, 'r') as results_file:
         return {line.rstrip() for line in results_file.readlines()}
 
     tempdir = tempfile.mkdtemp()
     tmp_output = os.path.join(tempdir, 'minimize-output1.txt')
-    run_result = self.run_pants(['minimize',
-                                 'testprojects::',
-                                 '--quiet',
-                                 '--minimize-output-file={0}'.format(tmp_output)])
+    run_result = self.run_pants(
+      ['minimize', 'testprojects::', '--quiet', f'--minimize-output-file={tmp_output}']
```

Comment:
Even though we glob against all of `testprojects`, with a chroot, it will just have the couple of folders declared in this targets BUILD.

Reply:
That's probably worth mentioning in a comment in the code, since it won't be clear to the casual reader. Or, can we glob against something more specific?